app/services/save-agendaitem-changes.js

```diff
@@ -0,0 +1,38 @@
+import Service, { inject as service } from '@ember/service';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+import { setNewPropertiesToModel, setModifiedOnAgendaOfAgendaitem } from 'frontend-kaleidos/utils/agendaitem-utils';
```

Comment:
Can we embed these utils in this service since it seems the only place where they are used.